src/zkevm_specs/evm/execution/error_Invalid_jump.py

```diff
@@ -0,0 +1,53 @@
+from ...util import FQ
+from ..instruction import Instruction, Transition
+from ..table import CallContextFieldTag
+from ..execution_state import ExecutionState
+from ..opcode import Opcode
+from ...util import N_BYTES_PROGRAM_COUNTER
+
+
+def invalid_jump(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+    # current executing op code must be JUMP or JUMPI
+    instruction.constrain_in(opcode, [FQ(Opcode.JUMP), FQ(Opcode.JUMPI)])
+    code_length = instruction.bytecode_length(instruction.curr.code_hash)
+    dest = instruction.stack_pop()
```

Comment:
When `JUMPI` we need to make sure the condition is `True`.

Reply:
good point!, also need to pop more one from stack which affects `rw_counter`, will update it.

Reply:
updated in [d2eb296](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/296/commits/d2eb2961e90e2c5331a1346c09f9ed58914394a9) if it is `jumpi`, pop one more `condition` from stack and constrain condition is zero. thanks!